src/python/pants/backend/javascript/subsystems/nodejs.py

```diff
@@ -556,12 +556,16 @@ async def prepare_corepack_tool(
                 None, ("corepack", "prepare", tool_spec if version else None, "--activate")
             ),
             description=f"Preparing configured {tool_description}.",
-            input_digest=request.input_digest,
+            # TODO: Given below, is there ever any need to supply an input_digest (e.g. package.json files) here?
+            # as far as I can see, the version is always resolved for the workspace and passed in as a CLI arg to corepack
+            # input_digest=request.input_digest,
```

Comment:
Yes, I believe supporting the default style of specifying package manager versions in `package.json` has real value. I think the issue is I accidentally only tested this with a `package.json` that specifies a pkg manager requirement at the build root, but it shouldn't be a requirement (as hinted by the passing of a working directory).